### ISSUE TITLE
Generated python client creates errors

### ISSUE THREAD

**Issue by grimpy** [Jumpscale/go-raml#259]
```
[/tmp] ipython2
Python 2.7.13 (default, Dec 21 2016, 07:16:46)
Type "copyright", "credits" or "license" for more information.

IPython 5.1.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import gridclient
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-de2ad0efc2eb> in <module>()
----> 1 import gridclient

/tmp/gridclient/__init__.py in <module>()
      5 from .BridgeCreate import BridgeCreate
      6 from .BridgeCreateSetting import BridgeCreateSetting
----> 7 from .CPUInfo import CPUInfo
      8 from .Cluster import Cluster
      9 from .ClusterCreate import ClusterCreate

/tmp/gridclient/CPUInfo.py in <module>()
      2 Auto-generated class for CPUInfo
      3 """
----> 4 from .float import float
      5 from .int32 import int32
      6

ImportError: No module named float
```

Also its not python3 compatible

```
In [1]: import gridclient
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-de2ad0efc2eb> in <module>()
----> 1 import gridclient

/tmp/gridclient/__init__.py in <module>()
      2
      3
----> 4 from .Bridge import Bridge
      5 from .BridgeCreate import BridgeCreate
      6 from .BridgeCreateSetting import BridgeCreateSetting

/tmp/gridclient/Bridge.py in <module>()
      4 from .EnumBridgeStatus import EnumBridgeStatus
      5
----> 6 from . import client_support
      7
      8

/tmp/gridclient/client_support.py in <module>()
      8 from uuid import UUID
      9 from enum import Enum
---> 10 from dateutil import parser
     11
     12

ModuleNotFoundError: No module named 'dateutil'
```

Generated with
/home/Jo/code/gopath/bin/go-raml client -l python --dir gridclient --ramlfile /home/Jo/code/gopath/src/github.com/g8os/grid/raml/api.raml

Raml file can be found at `https://github.com/g8os/grid/blob/nodes-api/raml/api.raml`

**Comment by iwanbk** [comment]
@grimpy fixed in 80dbc9d0f877162a97c6d46cfd4b3bd03b322a23 at `python-inheritance` branch.

Not fixed in separate branch because there are many changes regarding python class there.
That branch is almost finished, should be ready for PR today or tomorrow.

**Comment by iwanbk** [comment]
for 
```
ModuleNotFoundError: No module named 'dateutil'
```
we need to install `python-dateutil` package.
